sopel/modules/reddit.py

```diff
@@ -279,19 +325,23 @@ def subreddit_info(bot, trigger, match, commanded=False):
 
     try:
         s = r.subreddit(match)
-        s.subreddit_type
+        if match_lower not in ['all', 'popular']:
+            s.subreddit_type
     except prawcore.exceptions.Forbidden:
         bot.reply("r/" + match + " appears to be a private subreddit!")
         return plugin.NOLIMIT
     except prawcore.exceptions.NotFound:
         bot.reply("r/" + match + " appears to be a banned subreddit!")
         return plugin.NOLIMIT
 
+    if sorting in ['hot', 'new', 'top', 'controversial', 'gilded', 'rising', 'sticky', 'random']:
+        return subreddit_sorting(bot, trigger, s, sorting)
+
     link = "https://reddit.com/r/" + s.display_name
 
     created = get_time_created(bot, trigger, s.created_utc)
 
-    message = ('{link}{nsfw} | {subscribers} subscribers | '
+    message = ('[REDDIT] {link}{nsfw} | {subscribers} subscribers | '
```

Comment:
Since #1937 was merged, this is unnecessary.

```suggestion
    message = ('{link}{nsfw} | {subscribers} subscribers | '
```